definitions/infra-awsecscontainerinstance/golden_metrics.yml

```diff
@@ -0,0 +1,16 @@
+registeredCpu:
+  title: Registered CPUs
+  query:
+    select: average(provider.registeredCpu)
+    from: ComputeSample
```

Comment:
just curious but is there any reason to use sample in the golden metrics and Metric in the summaries?

Reply:
I followed the same approach used in `infra-awsecsservice` and `infra-awsecscluster`. Seems like this approach is used in all our cloud integrations. If you think I should follow different approach, I can update with suggested changes

Reply:
no its ok! I just wanted to know if there was a specific reason for it :D